lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -83,12 +83,25 @@ describe('NxCombobox', function() {
         expect(inputElement).toHaveValue('Foo');
       });
 
-  it('sets aria-expanded on the input to true when focused',
+  it('sets aria-expanded on the input according to if the input is focused and when an option is selected',
```

Comment:
This test name isn't very clear.  Since there are multiple ways for the relevant state change to happen, it might make sense to break it into several tests, but as an example one of those tests could be `it('sets aria-expanded to false when a dropdown item is selected by click'`.